js/render/draw_symbol.js

```diff
@@ -197,13 +203,15 @@ function drawSymbol(painter, layer, posMatrix, tile, bucket, bufferGroups, isTex
         gl.uniform1f(program.u_bearing, tr.bearing / 360 * 2 * Math.PI);
         gl.uniform1f(program.u_aspect_ratio, tr.width / tr.height);
 
+        painter.setDepthSublayer(1);
```

Comment:
Sublayers are used to let one part of a layer (antialiasing outline) be clipped by another (the main fill). Since the icons and text are never written to the depth buffer they can have the same depth sublayer. Having different ones shouldn't break anything but it isn't necessary.